### PR TITLE
Add charges to slip time that displays in UI

### PR DESCRIPTION
**Pull request by NathanielLow**
Adds charges to slip time and a UI element that displays them.

### CODE REVIEW

**Comment by NathanielLow** [comment]
[Slip Time Charges](https://trello.com/c/oeOY0N5L/65-slip-time-charges)

File: Assets/Scripts/UI/SlipTimeUi.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using SlipTime;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UI
+{
+    public class SlipTimeUi : MonoBehaviour
```
Comment (ProgrammingIncluded) [inline Assets/Scripts/UI/SlipTimeUi.cs]: `SlipTimeUI` is preferred here.

File: Assets/Scripts/UI/SlipTimeUi.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        public SlipTimeManager slipTimeManager;
+    
+        public Text slipTimeUiText;
```
Comment (ProgrammingIncluded) [inline Assets/Scripts/UI/SlipTimeUi.cs]: Please captilize the `I` as it is preferred here.

File: Assets/Scripts/UI/SlipTimeUi.cs (hunk 1 above)
Comment (NathanielLow) [reply Assets/Scripts/UI/SlipTimeUi.cs]: Fixed.

File: Assets/Scripts/UI/SlipTimeUi.cs (hunk 2 above)
Comment (NathanielLow) [reply Assets/Scripts/UI/SlipTimeUi.cs]: Fixed.

**Review by hirudan — APPROVED** [review]

**Pull request merged**